### PR TITLE
UC DOC - REST API Quickstart improvement

### PR DESCRIPTION
**Pull request by chartrandf**
- Remove gap bellow TOC items
- Move REST API - Quickstart first
- Improve content a bit

### CODE REVIEW

File: content/uc-doc/api_sdk/rest_api/quickstart.md [hunk 1]
```diff
@@ -82,20 +82,20 @@ To that end:
...
-1. Choose the `wazo-auth` service in the list of REST APIs
-2. In the top-right text box of the page (left to the "Explore" button), fill "token" with the
-   `rest-api-test:password`: those credentials are the ones from the Web Services Access you created
-   earlier.
-3. Go to the `POST /tokens` section and click on the yellow box to the right of the `body`
+1. Open `wazo-auth` spec in our [API Console](/documentation/console/authentication)
+2. In the first field, please enter your stack hostname (ex: `https://mystack.example`)
+3. In the last field, please enter `username:password` : those credentials are the ones from
+   the Web Services Access you created earlier.
+4. Go to the `POST /tokens` section and click on the yellow box to the right of the `body`
```
Comment (fblackburn1) [inline content/uc-doc/api_sdk/rest_api/quickstart.md:89]: To accept this answer we need to change the previous section that explain how to use the embedded swagger-ui  and add notice that no information will go through our server
The downside with the wazo-platform.org/api is that is not versioned compared to the embedded one

File: content/uc-doc/api_sdk/rest_api/quickstart.md [hunk 2]
```diff
@@ -58,53 +51,28 @@ Almost all REST APIs use encryption and are available via HTTPS. Unfortunately,
...
-   somewhere, in case you need it later.
-7. Copy-paste the `token` attribute in the top-right input box, replacing the
-   `rest-api-test:password`. Note that you don't need to click the Explore button to accept the
-   change of token.
-
-## Use the wazo-confd REST API
+2. Copy the URL `https://<YOUR_WAZO_IP>` and paste it in your browser.
+3. Accept the HTTPS certificate validation exception.
+4. Go back to the `API console`.
+5. Now you should see a list of sections for the authentication REST API, like `backends` or `token`
```
Comment (sduthil) [inline content/uc-doc/api_sdk/rest_api/quickstart.md:66]:
```suggestion
5. Now you should see the documentation for the `wazo-confd` API, and a list of sections for the configuration REST API, like `funckeys` or `users`
```

File: content/uc-doc/api_sdk/rest_api/quickstart.md [hunk 3]
```diff
@@ -58,53 +51,28 @@ Almost all REST APIs use encryption and are available via HTTPS. Unfortunately,
...
+
+## Use the Configuration REST API
 
 Now that we have an authentication token, we are ready to use the REST API.
```
Comment (sduthil) [inline content/uc-doc/api_sdk/rest_api/quickstart.md:70]: We don't have an authentication token at that point.

File: content/uc-doc/api_sdk/rest_api/quickstart.md [hunk 4]
```diff
@@ -58,53 +51,28 @@ Almost all REST APIs use encryption and are available via HTTPS. Unfortunately,
...
 
-1.  Click on wazo-confd in the left menu
-2.  Choose a REST API endpoint, like `users -> GET /users` and click `Try it out`
+1. Go to [Configuration - API Console](/documentation/console/configuration)
+2. In the top-left field, enter your stack hostname (ex: `https://mystack.example`)
+3. In the top-right field, enter your `username:password`  and click to `validate` to obtain a token
```
Comment (sduthil) [inline content/uc-doc/api_sdk/rest_api/quickstart.md:74]:
```suggestion
3. In the top-right field, enter your `username:password`  and click to `validate` to obtain a temporary token
```

**Review by sduthil — CHANGES_REQUESTED** [review]

**Review by sduthil — APPROVED** [review]

**Pull request merged**